tensorflow/lite/micro/micro_interpreter.h

```diff
@@ -110,6 +116,10 @@ class MicroInterpreter {
 
   TfLiteStatus initialization_status() const { return initialization_status_; }
 
+  size_t operators_size() const {
+    return model_->subgraphs()->Get(0)->operators()->size();
```

Comment:
Can you explain a bit about why the operator size is needed for LSTMs?

Reply:
its not required and will revert this change